clang/test/Interpreter/const.cpp

```diff
@@ -0,0 +1,29 @@
+// UNSUPPORTED: system-aix
+// RUN: cat %s | clang-repl | FileCheck %s
+// RUN: cat %s | clang-repl -Xcc -O2 | FileCheck %s
+
+extern "C" int printf(const char*, ...);
+
```

Comment:
Can you add a case for a file scope constant? We also should consider enabling that for C.

Reply:
`const A a(1);` is a file-scope constant, no? We don't need it for C because the special case in `LinkageComputer::getLVForNamespaceScopeDecl` only applies to C++ (my understanding is that `const` variables always have an identity in C).

Reply:
> const A a(1); is a file-scope constant, no?

Yes, missed that.